src/acrquery/azext_acrquery/_client_factory.py

```diff
@@ -0,0 +1,7 @@
+VERSION_2021_08_01_PREVIEW = "2023-08-01-preview"
```

Comment:
@CarolineNB Please create a new const that points to the latest API version [2023-01-01-preview](https://github.com/Azure/azure-rest-api-specs/tree/main/specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2023-01-01-preview).